hybrasyl/Objects/Monster.cs

```diff
@@ -310,6 +324,89 @@ public object Clone()
         {
             return this.MemberwiseClone();
         }
-    }
 
+        private class MonsterLoot
```

Comment:
This entire class looks like it should be static. Spawn related static objects are contained within the monolith class, which is designed to handle methods like this.